cdflow_commands/process.py

```diff
@@ -0,0 +1,11 @@
+from cdflow_commands.exceptions import UserFacingError
+from cdflow_commands.logger import logger
+
+from subprocess import CalledProcessError, check_call as _check_call
+
+
+def check_call(*args, **kwargs):
```

Comment:
I think this is fine, is it worth adding a really simple test for this method? Confirm it throws a usererror if `_check_call` returns in an error? Not sure how valuable it is or not but everything looks good otherwise

Reply:
Good point, I'll add one for the method.